readme.md

```diff
@@ -38,6 +38,7 @@ var favicons = require('favicons'),
         lang: "en-US"                   // Primary language for name and short_name
         background: "#fff",             // Background colour for flattened icons. `string`
         theme_color: "#fff",            // Theme color user for example in Android's task switcher. `string`
+        apple_status_bar_style: "black-translucent", // Style for Apple status bar: "black-translucent", "default", "black". `string`
```

Comment:
Would you mind switching to camel case? Snake case is only used for the ones that end up in `manifest.json`, which are styled this way.

Also please realign comments.